### PR TITLE
Update images on /desktop/features

### LINKED ISSUE #7256: Update the /desktop/features page with new images
Update [Desktop features](https://ubuntu.com/desktop/features) page baed on the [copy doc](https://docs.google.com/document/d/1nLHJlZ3igDwAAOUDP03wp4iKLxZts8110EA2ZOF5uaU/edit)

- [x] banner - update Ubuntu Software screenshot
- [x] office software - new libre office
- [x] web browsing - frankie and grace screenshot Peter to create
- [x] photo - new screenshot , icons for krita, gimp, blender
- [x] videos - use the youtube image
- [ ] gaming - sam to fix text

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update images on /desktop/features
- redid the wide desktop, email, libre office images.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/features
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the images look correct

## Issue / Card

Fixes #7256

### CODE REVIEW

File: templates/desktop/features.html [hunk 1]
```diff
@@ -64,7 +62,7 @@ <h2>A whole world of apps</h2>
     </div>
   </div>
   <div class="u-fixed-width">
-    <ul class="p-matrix is-trisected">
+    <ul class="p-matrix isp-strip--suru-half-and-half-reversed is-dark-trisected">
```
Comment (deadlight) [inline templates/desktop/features.html:65]: malformed class

File: templates/desktop/features.html [hunk 2]
```diff
@@ -394,7 +379,7 @@ <h2>Videos</h2>
     <div class="col-6 col-start-large-7 p-gaming__content">
       <div class="p-card--overlay">
         <h2>Gaming</h2>
-        <p>From Sudoku to shoot-em-ups, we&rsquo;ve got loads of games that&rsquo;ll keep you busy for hours.  There are thousands of games available for Ubuntu, including titles from the Unity and Steam platforms.  Pick from critically acclaimed titles such as Dota 2, Civilization V, Kerbal Space Program, Football Manager 2018 and Borderlands: The Pre-Sequel.</p>
+        <p>From Sudoku to first-person shooters, we’ve got loads of games that’ll keep you busy for hours.  There are thousands of games available for Ubuntu, including titles from the Unity and Steam platforms.  Pick from critically acclaimed titles such as Dota 2, Kerbal Space Program, Counter Strike: Global Offensive and Borderlands: The Pre-Sequel.</p>
```
Comment (deadlight) [inline templates/desktop/features.html:382]: Down with the kids ;)

File: templates/desktop/features.html (hunk 1 above)
Comment (pmahnke) [reply templates/desktop/features.html]: I must have slipped... fixed

File: templates/desktop/features.html (hunk 2 above)
Comment (pmahnke) [reply templates/desktop/features.html:382]: Well @SirSamTumless is.

**Pull request merged**